package.json

```diff
@@ -140,5 +141,15 @@
       "cp -af /usr/lib/PSAMMEAD-COMPONENT-LIBRARY/bake-scripts/. /etc/bake-scripts/PSAMMEAD-COMPONENT-LIBRARY",
       "chmod +x /etc/bake-scripts/PSAMMEAD-COMPONENT-LIBRARY/*"
     ]
+  },
+  "lint-staged": {
+    "*.js": [
```

Comment:
This will need to run against the same file extensions that we current lint i.e. `*.{js,jsx,json}` at the momtn this only runs against `.js` files.

We also do not current use the auto-fix option so this should be removed.

Something like 
```
"lint-staged": {
    "*.{js,jsx,json}": "eslint"
  }
```

Reply:
Done